### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by josteinaj**
This is the docker stuff merged into release/1.7.0 branch.

Versions are read from the same versions.yml file as the ansible stuff.

I've set up docker hub to automatically build the master branch as "latest" (i.e. default), as well as images for each tag: https://hub.docker.com/r/snaekobbi/system/

### CODE REVIEW

**Comment by dkager** [comment]
This is awesome work.

File: README.md [hunk 1]
```diff
@@ -39,3 +45,92 @@ Engine, enter "http://localhost:8181/ws".
...
+Successfully built 72ca1a0390e8
+```
+
+In this case, `72ca1a0390e8` is the image ID.
+
+@josteinaj has uploaded pre-built images to https://hub.docker.com/r/josteinaj/snaekobbi/ which
+means you can use those instead by simply referring to `josteinaj/snaekobbi` instead of
+building your own image. That image will be used in the following commands.
+
+**TODO**: set up automatic building of `snaekobbi/system` images instead.
```
Comment (bertfrees) [inline README.md]: This should happen automatically as soon as this is merged, right?

Comment (josteinaj) [inline README.md]: Yes, just pushed an edit to the README.

Comment (josteinaj) [inline README.md]: Well... as soon as it is merged into `master`; this PR merges into the `release/1.7.0` branch since `master` is a bit behind.

**Pull request merged**